### ISSUE TITLE
Errors when using a lot of consumers

### ISSUE THREAD

**Issue by purple-emily** [Gabisonfire/knightcrawler#71]
These errors stabilise eventually, or at least they did when I was using a 30/30 ratio.

But I bumped to a 35/35 ratio and it doesn't seem to stabilise. This is AFTER #68 was merged:

```
JOB_CONCURRENCY=35
CONSUMER_REPLICAS=35
```

The consumers crash with this error:

```
Node.js v20.11.0
(node:1) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.      
(Use `node --trace-deprecation ...` to show where the warning was created)
{"level":30,"time":1707139610848,"pid":1,"hostname":"9393772d37e8","msg":"Trackers updated at 1707139610848: 102 trackers"}
Failed syncing database:  Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}
```

**Comment by purple-emily** [comment]
```
consumer-6   | Failed syncing database:  Error
consumer-6   |     at qE.run (/app/dist/index.cjs:199:117541)
consumer-6   |     at /app/dist/index.cjs:199:235551
consumer-6   |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
consumer-6   |     at async Ere.showIndex (/app/dist/index.cjs:182:12425)
consumer-6   |     at async file.sync (/app/dist/index.cjs:181:21232)
consumer-6   |     at async t.sync (/app/dist/index.cjs:199:237026)
consumer-6   |     at async /app/dist/index.cjs:803:1578 {
consumer-6   |   name: 'SequelizeDatabaseError',
consumer-6   |   parent: error: cache lookup failed for attribute 1 of relation 87103
consumer-6   |       at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
consumer-6   |       at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
consumer-6   |       at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)       
consumer-6   |       at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)  
consumer-6   |       at Socket.emit (node:events:518:28)
consumer-6   |       at addChunk (node:internal/streams/readable:559:12)
consumer-6   |       at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)     
consumer-6   |       at Readable.push (node:internal/streams/readable:390:5)
consumer-6   |       at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
consumer-6   |     length: 116,
consumer-6   |     severity: 'ERROR',
consumer-6   |     code: 'XX000',
consumer-6   |     detail: undefined,
consumer-6   |     hint: undefined,
consumer-6   |     position: undefined,
consumer-6   |     internalPosition: undefined,
consumer-6   |     internalQuery: undefined,
consumer-6   |     where: undefined,
consumer-6   |     schema: undefined,
consumer-6   |     table: undefined,
consumer-6   |     column: undefined,
consumer-6   |     dataType: undefined,
consumer-6   |     constraint: undefined,
consumer-6   |     file: 'lsyscache.c',
consumer-6   |     line: '1008',
consumer-6   |     routine: 'get_attoptions',
consumer-6   |     sql: "SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'files' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;",
consumer-6   |     parameters: undefined
consumer-6   |   },
consumer-6   |   original: error: cache lookup failed for attribute 1 of relation 87103
consumer-6   |       at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
consumer-6   |       at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
consumer-6   |       at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)       
consumer-6   |       at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)  
consumer-6   |       at Socket.emit (node:events:518:28)
consumer-6   |       at addChunk (node:internal/streams/readable:559:12)
consumer-6   |       at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)     
consumer-6   |       at Readable.push (node:internal/streams/readable:390:5)
consumer-6   |       at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
consumer-6   |     length: 116,
consumer-6   |     severity: 'ERROR',
consumer-6   |     code: 'XX000',
consumer-6   |     detail: undefined,
consumer-6   |     hint: undefined,
consumer-6   |     position: undefined,
consumer-6   |     internalPosition: undefined,
consumer-6   |     internalQuery: undefined,
consumer-6   |     where: undefined,
consumer-6   |     schema: undefined,
consumer-6   |     table: undefined,
consumer-6   |     column: undefined,
consumer-6   |     dataType: undefined,
consumer-6   |     constraint: undefined,
consumer-6   |     file: 'lsyscache.c',
consumer-6   |     line: '1008',
consumer-6   |     routine: 'get_attoptions',
consumer-6   |     sql: "SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'files' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;",
consumer-6   |     parameters: undefined
consumer-6   |   },
consumer-6   |   sql: "SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'files' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;",
consumer-6   |   parameters: {}
consumer-6   | }
consumer-6   | node:internal/process/promises:289
consumer-6   |             triggerUncaughtException(err, true /* fromPromise */);
consumer-6   |             ^
consumer-6   |
consumer-6   | Error
consumer-6   |     at qE.run (/app/dist/index.cjs:199:117541)
consumer-6   |     at /app/dist/index.cjs:199:235551
consumer-6   |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
consumer-6   |     at async Ere.showIndex (/app/dist/index.cjs:182:12425)
consumer-6   |     at async file.sync (/app/dist/index.cjs:181:21232)
consumer-6   |     at async t.sync (/app/dist/index.cjs:199:237026)
consumer-6   |     at async /app/dist/index.cjs:803:1578 {
consumer-6   |   name: 'SequelizeDatabaseError',
consumer-6   |   parent: error: cache lookup failed for attribute 1 of relation 87103
consumer-6   |       at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
consumer-6   |       at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
consumer-6   |       at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)       
consumer-6   |       at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)  
consumer-6   |       at Socket.emit (node:events:518:28)
consumer-6   |       at addChunk (node:internal/streams/readable:559:12)
consumer-6   |       at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)     
consumer-6   |       at Readable.push (node:internal/streams/readable:390:5)
consumer-6   |       at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
consumer-6   |     length: 116,
consumer-6   |     severity: 'ERROR',
consumer-6   |     code: 'XX000',
consumer-6   |     detail: undefined,
consumer-6   |     hint: undefined,
consumer-6   |     position: undefined,
consumer-6   |     internalPosition: undefined,
consumer-6   |     internalQuery: undefined,
consumer-6   |     where: undefined,
consumer-6   |     schema: undefined,
consumer-6   |     table: undefined,
consumer-6   |     column: undefined,
consumer-6   |     dataType: undefined,
consumer-6   |     constraint: undefined,
consumer-6   |     file: 'lsyscache.c',
consumer-6   |     line: '1008',
consumer-6   |     routine: 'get_attoptions',
consumer-6   |     sql: "SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'files' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;",
consumer-6   |     parameters: undefined
consumer-6   |   },
consumer-6   |   original: error: cache lookup failed for attribute 1 of relation 87103
consumer-6   |       at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
consumer-6   |       at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
consumer-6   |       at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)       
consumer-6   |       at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)  
consumer-6   |       at Socket.emit (node:events:518:28)
consumer-6   |       at addChunk (node:internal/streams/readable:559:12)
consumer-6   |       at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)     
consumer-6   |       at Readable.push (node:internal/streams/readable:390:5)
consumer-6   |       at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
consumer-6   |     length: 116,
consumer-6   |     severity: 'ERROR',
consumer-6   |     code: 'XX000',
consumer-6   |     detail: undefined,
consumer-6   |     hint: undefined,
consumer-6   |     position: undefined,
consumer-6   |     internalPosition: undefined,
consumer-6   |     internalQuery: undefined,
consumer-6   |     where: undefined,
consumer-6   |     schema: undefined,
consumer-6   |     table: undefined,
consumer-6   |     column: undefined,
consumer-6   |     dataType: undefined,
consumer-6   |     constraint: undefined,
consumer-6   |     file: 'lsyscache.c',
consumer-6   |     line: '1008',
consumer-6   |     routine: 'get_attoptions',
consumer-6   |     sql: "SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'files' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;",
consumer-6   |     parameters: undefined
consumer-6   |   },
consumer-6   |   sql: "SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'files' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;",
consumer-6   |   parameters: {}
consumer-6   | }
consumer-6   |
consumer-6   | Node.js v20.11.0
```

I don't know if this is the same set of error messages.
6 -> 12 replicas and they repeat this loop cylce around 6 times. They restart themselves, crash again, then eventually stabilise.

**Comment by funkypenguin** [comment]
Likewise, I have a bunch:

```
Failed syncing database:  Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
Failed syncing database:  Error
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
  name: 'SequelizeUnknownConstraintError',
    at qE.run (/app/dist/index.cjs:199:117541)
Failed syncing database:  Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
    internalPosition: undefined,
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
    internalQuery: undefined,
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at Readable.push (node:internal/streams/readable:390:5)
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
Failed syncing database:  Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
  },
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
      at Readable.push (node:internal/streams/readable:390:5)
    length: 137,
Failed syncing database:  Error
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    severity: 'ERROR',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}

Node.js v20.11.0
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
    length: 137,
    code: '42704',
Failed syncing database:  Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
    routine: 'ATExecDropConstraint',
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
    severity: 'ERROR',
    detail: undefined,
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    parameters: undefined
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    detail: undefined,
    position: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
  },
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    hint: undefined,
    internalPosition: undefined,
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    length: 137,
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
    position: undefined,
    internalQuery: undefined,
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
    severity: 'ERROR',
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    where: undefined,
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    code: '42704',
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
    internalQuery: undefined,
    schema: undefined,
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}

Node.js v20.11.0
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}

Node.js v20.11.0
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    where: undefined,
    table: undefined,
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}

Node.js v20.11.0
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}

Node.js v20.11.0
    schema: undefined,
    table: undefined,
    column: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}

Node.js v20.11.0
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file.sync (/app/dist/index.cjs:181:21117)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeUnknownConstraintError',
  parent: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  original: error: constraint "files_infoHash_fkey" of relation "files" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 137,
    severity: 'ERROR',
    code: '42704',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'tablecmds.c',
    line: '12053',
    routine: 'ATExecDropConstraint',
    sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
    parameters: undefined
  },
  sql: 'ALTER TABLE "files" DROP CONSTRAINT "files_infoHash_fkey"',
  parameters: {},
  constraint: 'files_infoHash_fkey',
  fields: undefined,
  table: 'files'
}

Node.js v20.11.0
```

**Comment by funkypenguin** [comment]
A nice, concise one:

```
Defaulted container "torrentio-consumer" out of: torrentio-consumer, gluetun
(node:1) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
{"level":30,"time":1707140791187,"pid":1,"hostname":"torrentio-consumer-5c847b77db-b5r88","msg":"Trackers updated at 1707140791187: 102 trackers"}
Failed syncing database:  Error
    at qE.run (/app/dist/index.cjs:199:117541)
    at /app/dist/index.cjs:199:235551
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Ere.showIndex (/app/dist/index.cjs:182:12425)
    at async ingested_torrent.sync (/app/dist/index.cjs:181:21232)
    at async t.sync (/app/dist/index.cjs:199:237026)
    at async /app/dist/index.cjs:803:1578 {
  name: 'SequelizeDatabaseError',
  parent: error: cache lookup failed for attribute 1 of relation 27977
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
      at Readable.push (node:internal/streams/readable:390:5)
```

**Comment by purple-emily** [comment]
There is also this error:

```
consumer-1  | Failed syncing database:  TypeError: Cannot read properties of null (reading '1')
consumer-1  |     at qE.run (/app/dist/index.cjs:199:118777)
consumer-1  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) 
consumer-1  |     at async /app/dist/index.cjs:199:235543
consumer-1  |     at async Ere.showIndex (/app/dist/index.cjs:182:12425)
consumer-1  |     at async file.sync (/app/dist/index.cjs:181:21232)
consumer-1  |     at async t.sync (/app/dist/index.cjs:199:237026)
consumer-1  |     at async /app/dist/index.cjs:803:1578
consumer-1  | /app/dist/index.cjs:199
```

**Comment by purple-emily** [comment]
HERES A POSSIBLE ANSWER:

```
consumer-25  | (node:1) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
consumer-25  | (Use `node --trace-deprecation ...` to show where the warning was created)       
consumer-25  | {"level":30,"time":1707148109896,"pid":1,"hostname":"da2e1415fe9f","msg":"Trackers updated at 1707148109896: 102 trackers"}
consumer-25  | Failed syncing database:  remaining connection slots are reserved for roles with the SUPERUSER attribute
consumer-25  | /app/dist/index.cjs:199
```

**Comment by FunkeCoder23** [comment]
This is with a rebuilt image after #68?
The only reason I ask is that the error message should be much smaller due to the change to print `error.message` instead of the whole error object.
And that alter command looks like what that PR defaulted to false. 

Can you run `docker compose build` again and down/up your containers and see if the error messages shorten?

**Comment by purple-emily** [comment]
@FunkeCoder23 I beleive so. I will reclone and try again just to be sure

**Comment by funkypenguin** [comment]
Since applying https://github.com/Gabisonfire/knightcrawler/pull/82, I've not seen these errors!

**Comment by purple-emily** [comment]
Assuming this is resolved by both #68 and #82. If anyone sees any more errors like this please open a new issue
